srcpkgs/looking-glass/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'looking-glass'
+pkgname=looking-glass
+version=5.0.1
+revision=1
+archs="x86_64"
+wrksrc="$pkgname-B$version/client"
+build_style=cmake
+configure_args="-DENABLE_BACKTRACE=no"
+makedepends="cmake make gcc libglapi libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel pkg-config libX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp libwaylandpp-devel wayland-protocols dejavu-fonts-ttf libXScrnSaver-devel MesaLib-devel"
```

Comment:
```
cmake make gcc
```
already included in the build chroot, don't specify them (gcc: by default; make: not needed, void's cmake defaults to ninja; cmake: already included by the build style)

---
```
pkg-config
```
should be in `hostmakedepends`

---
```
libglapi
```
covered by `MesaLib-devel`

---
```
libwaylandpp dejavu-fonts-ttf
```
are these actually needed at build time?

---
```
libglib-devel nettle-devel gmp-devel fontconfig-devel spice-devel ibX11-devel libXfixes-devel libXi-devel libXinerama-devel libXcursor-devel libXpresent-devel libxkbcommon-devel libwaylandpp-devel libXScrnSaver-devel MesaLib-devel
```
the rest are probably fine